jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsSdkClient.kt

```diff
@@ -39,7 +43,7 @@ class AwsSdkClient : Disposable {
     }
 
     companion object {
-        private val LOG = Logger.getInstance(AwsSdkClient::class.java)
+        private val LOG = Logger.getInstance(DefaultAwsSdkClient::class.java)
 
         fun getInstance(): AwsSdkClient = ServiceManager.getService(AwsSdkClient::class.java)
```

Comment:
You should be able to move this up into the interface and reduce the blast radius of files changed.

Reply:
The next iteration removes the need to mock AwsSdkClient, so I removed the interface and changed the class name back to AwsSdkClient.